dynatademand/schemas/request/body/create_project.json

```diff
@@ -118,12 +132,7 @@
           "deliveryType": {
             "type": "string",
             "default": "BALANCED",
-            "description": "The plan on how responses will flow into the survey.",
-            "enum": [
```

Comment:
Why delete this?

Reply:
It is been replaced with string and validated for the input. The enum values are no longer being used in the implementation.